packages/machine-emulator-tools/README.md

```diff
@@ -0,0 +1,33 @@
+# Machine emulator tools packaging
+
+The [machine-emulator-tools](https://github.com/cartesi/machine-emulator-tools) is a set of tools that need to be installed in the root filesystem of a Cartesi DApp.
+
+It is currently distributed in binary format compiled for the `linux/riscv64` architecture and available as a `.tar.gz` file in the [GitHub releases](https://github.com/cartesi/machine-emulator-tools/releases).
+
+However the most convenient way to add this to the user application, which uses Docker as a build system, was to add a `COPY --from` directive, like the one below:
+
+```dockerfile
+COPY --from=cartesi/machine-emulator-tools:0.10.0 / /
+```
+
+While `cartesi` doesn't provide such image, Sunodo will provide the image under the `ghcr.io/sunodo` organization.
+
+So the application should include the following:
+
+```dockerfile
+COPY --from=ghcr.io/sunodo/machine-emulator-tools:0.10.0-ubuntu22.04 / /
+```
+
+Only `ubuntu 22.04` is currently supported. We might add other distributions in the future.
```

Comment:
We could start with `debian:bullseye` even if wee need to compile server-manager on a sunodo's fork while it's not upstream.

The plan is to run every service from cartesi-node AND server-manager on the same VM, for that, we need it to follow `debian:bullseye`, since that's what's being used on cartesi-node build from the cartei/rollups repository.

Reply:
The tools is another issue. This goes into the user application.
The user application can use any distro that has the tools compile for.
And it's not only compilation, its init script has to be compatible with all supported distributions.

Reply:
This moved to PR https://github.com/sunodo/sunodo/pull/22